packages/headless/src/controllers/breadcrumb-manager/headless-breadcrumb-manager.ts

```diff
@@ -135,37 +145,76 @@ export const buildBreadcrumbManager = (
         dateFacetBreadcrumbs: getDateFacetBreadcrumbs(),
       };
     },
+    /**
+     * Determines if there's any available breadcrumbs, or active facets.
+     */
     hasBreadcrumbs,
+    /**
+     * Allows to deselect and clear all facet filters.
+     */
     deselectAll: () => {
       dispatch(deselectAllFacets());
       dispatch(executeSearch(logClearBreadcrumbs()));
     },
   };
 };
 
+/**
+ * Represent a generic Breadcrumb type.
+ */
 export interface Breadcrumb<T extends BaseFacetValue> {
+  /**
+   * The field on which the underlying facet is configured.
+   */
   field: string;
+  /**
+   * The list of facet values currently active and selected.
+   */
   values: BreadcrumbValue<T>[];
 }
 
+/**
+ * Represent a generic Breadcrumb value type.
+ *
+ * Can either be a @type {FacetBreadcrumb}, @type {NumericFacetBreadcrumb}, @type {DateFacetBreadcrumb}, @type {CategoryFacetBreadcrumb}
+ */
 export type BreadcrumbValue<T extends BaseFacetValue> = {
+  /**
+   * The underlying facet value linked to this breadcrumb.
+   */
   value: T;
+  /**
+   * Allow to deselect and clear the corresponding facet filter.
+   */
   deselect: () => void;
 };
 
+/**
+ * Represent a Breadcrumb for specific facet.
+ */
 export type FacetBreadcrumb = Breadcrumb<FacetValue>;
+/**
+ * Represent a Breadcrumb for a numerical facet.
+ */
 export type NumericFacetBreadcrumb = Breadcrumb<NumericFacetValue>;
+/**
+ * Represent a Breadcrumb for a date facet.
+ */
 export type DateFacetBreadcrumb = Breadcrumb<DateFacetValue>;
+/**
+ * Represent a breadcrumb for a category facet.
+ */
 export interface CategoryFacetBreadcrumb {
+  /**
+   * The field on which the underlying facet is configured.
+   */
   field: string;
+  /**
+   * The complete path of the underlying category facet value.
+   */
   path: CategoryFacetValue[];
+  /**
+   * Allow to deselect and clear the corresponding facet filter.
+   */
   deselect: () => void;
 }
-
-export type BreadcrumbField = Pick<
```

Comment:
I did a quick grep, and there's no reference in any package.  

Useless artefact or actually useful type in some place ?

Reply:
Ho, that's my bad. I added it during the breadcrumb styling PR, but forgot to remove it because it isn't useful in the end. You can remove it